src/trustymail/trustymail.py

```diff
@@ -459,7 +468,7 @@ def spf_scan(resolver, domain):
 
 def parse_dmarc_report_uri(uri):
     """
-    Parses a DMARC Reporting (i.e. ``rua``/``ruf)`` URI
+    Parse a DMARC Reporting (i.e. ``rua``/``ruf)`` URI.
```

Comment:
Fix a typo while we are here:
```suggestion
    Parse a DMARC Reporting (i.e. ``rua``/``ruf``) URI.
```